Sources/XCTest/TestFiltering.swift

```diff
@@ -42,12 +37,12 @@ internal struct TestFiltering {
 
     static func filterTests(entries: [XCTestCaseEntry], filter: TestFilter) -> [XCTestCaseEntry] {
         return entries
-            .map({ testCase, tests in
-                return (testCase, tests.filter({ filter(testCase, $0.0) }))
-            })
-            .filter({ testCase, tests in
+            .map { testCase, tests in
+                return (testCase, tests.filter { filter(testCase, $0.0) } )
+            }
+            .filter { testCase, tests in
                 return !tests.isEmpty
-            })
+            }
```

Comment:
@briancroom Same here. I don't have a strong opinion, I think either way of writing it is fine.